plugins/lookup/passwordstore.py

```diff
@@ -32,6 +32,13 @@
         description: Overwrite the password if it does already exist.
         type: bool
         default: 'no'
+      umask:
+        description:
+          - Sets the umask for the created .gpg files. The first octed must be greater than 3 (user readable). Note
+            pass' default value is '077'.
```

Comment:
I think it's better to use two paragraphs here:
```suggestion
          - Sets the umask for the created .gpg files. The first octed must be greater than 3 (user readable).
          - Note pass' default value is C(077).
```
(For `C(...)` see https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation)

Reply:
thanks for this, i was unsure about the syntax. this is full yaml, right?